markdown-syntax.md

```diff
@@ -43,15 +43,15 @@ the content, manipulate, then put it back together.
 The images will now be added as below.
 
 ```html
-<div class="w4 h4 bg-center cover br-100 center"
+<div class="w4 h4 bg-center cover br-100 center mt1"
```

Comment:
@katbow result looks good. 👍 
_however_ for _future_ reference: we tend to apply margin _bottom_ to the element _above_ to push things down instead of margin top.